### PR TITLE
docs: device profile operation guide

### PR DESCRIPTION
**Pull request by reubenmiller**
## Proposed changes

Add device profile operation guide for users (with screenshots)

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvement (general improvements like code refactoring that doesn't explicitly fix a bug or add any new functionality)
- [x] Documentation Update (if none of the other choices apply)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Paste Link to the issue
<br/>

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTOR-LICENSE-AGREEMENT.md) (in all commits with git commit -s)
- [ ] I ran `cargo fmt` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I used `cargo clippy` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: docs/src/operate/c8y/device-profiles.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+title: Device Profiles
+tags: [Operate, Device Profile, Firmware Management, Software Management, Configuration Management]
+description: Managing device profiles on devices
+---
+
+## Management
+
+%%te%% enables you to manage a the firmware, software and configuration for a device in a single operation which makes it suitable for managing a large fleet of devices. The following sections detail how to create and assign a device profile to a device. Using Cumulocity IoT, you can either assign a device profile using a single operation (described on this page), or by using a [bulk operation](https://cumulocity.com/docs/device-management-application/monitoring-and-controlling-devices/#to-add-a-bulk-operation) to assign it to a fleet of devices.
```
Comment (gligorisaev) [inline docs/src/operate/c8y/device-profiles.md:9]: Small typo, the "a" 
%%te%% enables you to manage the firmware ....

File: docs/src/operate/c8y/device-profiles.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    :::note
+    Currently device profiles don't support/show the software type field in the UI. If you are using more than 1 software type on a device, then you will need to encode the software type into the software version, e.g. `1.2.3::apt`.
+    :::
+
+### To install a device profile on a device {#install}
+
+The following procedure describes the steps involved to assign a device profile to a device.
+
+:::tip
+Only previously created device profile can be selected. If you don't see any device profiles in the dropdown menu, then make sure you add a device profile using [these instructions](#add-to-repo).
```
Comment (albinsuresh) [inline docs/src/operate/c8y/device-profiles.md:50]:
```suggestion
Only previously created device profiles can be selected. If you don't see any device profiles in the dropdown menu, then make sure you add a device profile using [these instructions](#add-to-repo).
```

**Review by albinsuresh — APPROVED** [review]
LGTM.

**Review by didier-wenzek — APPROVED** [review]
Approved

**Pull request merged**